src/fivetran-destination/src/destination/dml.rs

```diff
@@ -231,120 +231,83 @@ async fn load_file(file_config: &FileConfig, path: &str) -> Result<AsyncFileRead
     Ok(file)
 }
 
+/// `DELETE` and then `INSERT` for all of the records in `replace_files` based on primary key.
```

Comment:
The description of the PR says that we no longer use `DELETE` and `INSERT`, am I missing something?